commands/translate.ts

```diff
@@ -70,21 +70,54 @@ export const translate = async (
     // Probably a language code, but if not, reply with an error.
     language = split[1].toLowerCase();
   } else {
-    return translateIBM(text, "", "en-fr").then((translation) => {
+    return translateIBM(text, "french").then((translation) => {
```

Comment:
@bobfri let's change this back: since the usual translation is en->fr, and sometimes shorter sentences can't get the source language detected, it may be better to hardcode the model to `en-fr`

Reply:
there is no problem i'll change it back, I get the point about hardcoding it for short sentence, i had changed it since i had read through the api and i found it better to have it autodecte the language, so no matter the language the default would be french and you wouldn't hog up characters translating it twice (if you do !translate first then !translate french) since characterrs are limited per month